Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN go mod download
 # build
 COPY cmd cmd
 COPY pkg pkg
-RUN go build -o /remediator cmd/remediator/app.go
+RUN go build -o /remediator cmd/remediator/*.go
```

Comment:
why do we need to build multiple ?

Reply:
because now there are two files in there `app.go` and `policy.go`

Reply:
I can move it to its own package I guess

`policy/policy.go`